pennylane/devices/default_tensor.py

```diff
@@ -352,8 +352,8 @@ def circuit(phi, depth, num_qubits):
             The execution time for this circuit with the above parameters is around 0.8 seconds on a standard laptop.
 
             The tensor network method can be faster than MPS and state vector methods in some cases.
-            As a comparison, the time for the exact calculation of the same circuit with the MPS method and with the ``default.qubit``
-            device is about three orders of magnitude slower.
+            As a comparison, the time for the exact calculation (i.e., with ``max_bond_dim = None``) of the same circuit using the MPS method with the ``default.qubit``
+            device is approximately three orders of magnitude slower.
```

Comment:
```suggestion
            As a comparison, the time for the exact calculation (i.e., with ``max_bond_dim = None``) of the same circuit with the `default.tensor` device using the MPS method is approximately three orders of magnitude slower. Using the ``default.qubit`` device also results in a much slower simulation.
```

Reply:
Oh I see, it was not about comparing the default.qubit vs MPS method, but both of them agains the TN method!! I like your suggestion!